### PR TITLE
Shrink k8gb role to what is really required

### PR DESCRIPTION
**Pull request by k0da**
fixes https://github.com/AbsaOSS/k8gb/issues/330

Signed-off-by: Dinar Valeev <dinar.valeev@absa.africa>

### CODE REVIEW

File: chart/k8gb/templates/role.yaml [hunk 1]
```diff
@@ -7,54 +7,11 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - pods
-  - nodes
   - services
-  - services/finalizers
   - endpoints
```
Comment (ytsarev) [inline chart/k8gb/templates/role.yaml:11]: For Endpoints we probably need only `get` verb

File: chart/k8gb/templates/role.yaml [hunk 2]
```diff
@@ -7,54 +7,11 @@ rules:
 - apiGroups:
   - ""
   resources:
-  - pods
-  - nodes
   - services
```
Comment (ytsarev) [inline chart/k8gb/templates/role.yaml:10]: Only `get` for Services

**Review by ytsarev — CHANGES_REQUESTED** [review]

File: chart/k8gb/templates/role.yaml (hunk 1 above)
Comment (k0da) [reply chart/k8gb/templates/role.yaml:11]: makes sense

File: chart/k8gb/templates/role.yaml (hunk 2 above)
Comment (k0da) [reply chart/k8gb/templates/role.yaml:10]: Also dropped events, so amended

**Review by ytsarev — APPROVED** [review]

**Pull request merged**